src/components/global/ContactFormModal.jsx

```diff
@@ -15,27 +16,34 @@ import emailjs from '@emailjs/browser';
 import { PropTypes } from 'prop-types';
 
 const ContactFormModal = ({ showContactFormModal, setShowContactFormModal }) => {
+  const [messageSuccess, setMessageSuccess] = useState(false);
+  const [messageFailure, setMessageFailure] = useState(false);
   const form = useRef();
 
   const handleClose = () => {
```

Comment:
I'm not positive about this as react state changes being asynchronous, but IF it runs the `setShowContactFormModal(false)` first - then it won't be able to do the other setter functions as the component will be unmounted and they will cease to exist.

I think you should have the `setShowContactFormModal(false)` last to combat this.